src/components/Scorecards/ScorecardDetailsPage/ScorecardFilterCard/ScorecardFilterCard.tsx

```diff
@@ -38,6 +38,7 @@ import {
   defaultGroupRefContext,
   defaultSystemRefContext,
 } from '../../../../utils/ComponentUtils';
+import {EntityFilterGroup} from "../../../../filters";
```

Comment:
nit:
```suggestion
import { EntityFilterGroup } from "../../../../filters";
```

Reply:
I thought we did??

Reply:
I see that we have `eslint` set up, but apparently it doesn't care about this particular thing. I honestly never remember what is enforced by prettier vs eslint so not sure exactly why this didn't get caught without investigating.